Assets/Scripts/Scoreboard/Scoreboard.cs

```diff
@@ -51,11 +60,14 @@ void Start()
 
     public void SetupPoster(Player player, string subtitle)
     {
-        background.color = player.playerIdentity.color;
+        wantedPoster.GetComponent<Image>().color = player.playerIdentity.color;
         this.subtitle.text = subtitle;
         this.player = player;
+        progressPoster.GetComponent<Image>().color = player.playerIdentity.color;
+        progressDescription.text = player.playerIdentity.playerName;
 
         scoreboardManager.ShowNextCrime += NextStep;
+        scoreboardManager.DisplayProgress += ShowProgress;
```

Comment:
The difference between Display and Show is... what, exactly? Some of these names could use a do-over.

Reply:
Has been updated 👍